packages/suite/src/views/wallet/receive/components/UsedAddresses/index.tsx

```diff
@@ -122,25 +122,33 @@ const Item = ({ addr, symbol, onClick, metadataPayload, index }: ItemProps) => {
     const fresh = addr.transfers < 1;
     const address = addr.address.substring(0, 20);
 
+    // prevent re-rendering of MetadataLabeling when isHovered state changes
+    const MemoizedMetadataLabeling = useMemo(
```

Comment:
shouldn't `useMemo` be declared inside `MetadataLabeling` component?

Reply:
yes!